examples/typescript/src/_routes.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import asyncComponent from '@jaredpalmer/after/asyncComponent';
+import Home from './Home';
+
+export default [
```

Comment:
We need types for this array.

Reply:
If i recall there is something in react-router-dom called RouteProps. This would be RouteProps[] (i think)